organisms/organisation-unit-tree/new-features/select-by-group-level.feature

```diff
@@ -0,0 +1,56 @@
+# Generic
+
+Feature: Selection of descendant levels and/or groups inside an org unit
+
+  Background:
+    Given an org unit tree with root org unit 'Sierra Leone'
+    And descendants inside 'Sierra Leone' with multiple groups and levels
+
+  Scenario: Selection of a level or group inside an org unit
+    Given the user has opened the context menu 'Sierra Leone'
+    When the user selects a level or group type from the 'Select by >' menu
+    Then all org units of that level or group are selected as a single selection
+    And 'Sierra Leone' org unit visually indicates the selection made
+
+  Scenario: Hide non-available groups and levels in 'Select by' menus
+    Given the user has opened the context menu for 'Sierra Leone'
+    Then only groups that are present in descendants should appear in 'Select by group >' sub-menu
+    And only levels that are present in descendants should appear in 'Select by level >' sub-menu
+
+  Scenario: Deselection of a level or group inside an org unit
```

Comment:
The selection of groups/levels scenario is quite detailed while this one is less specific about what happens.
We should make that consistent. 

After having read both more and less specific scenarios, I do like the less specific one way more! Because behavior shouldn't know anything about the design (context menu is already design I guess, should be totally fine to say "When the user selects the 'Select by group' option"), but we might have to experiment here a bit

Reply:
Yes, that's true. Defining the selection as taking place through the context menu also means this .feature file becomes obsolete if we used a different design/component (for example a popover or a hover button). I agree with your point about the less specific scenarios being more successful. I've simplified the selection scenario (and deselect too, a little) in both group and level files.